Makefile

```
@@ -19,7 +19,7 @@ train-nlu:
 	python3 -m rasa_nlu.train -c nlu_tensorflow.yml --fixed_model_name current --data data/nlu/ -o models --project nlu --verbose
 
 train-core:
-	python3 -m rasa_core.train -d domain.yml -s data/core -c policy.yml --debug -o models/dialogue
+	python3 -m rasa_core.train -d domain.yml -s data/core -c policy.yml --debug -o models/dialogue --augmentation 0
```

```suggestion
	python3 -m rasa_core.train -d domain.yml -s data/core -c policy.yml --debug -o models/dialogue 
```

we need the augmentation to stay for keras -- the behaviour of the augmented memo policy will be fixed with the latest core release anyways, and i'll be updating everything to that next week